compiler/llvm/clangUtil.cpp

```diff
@@ -4709,9 +4709,21 @@ static void stripPtxDebugDirective(const std::string& artifactFilename) {
 
 }
 
+static void setPATH(std::string PATH) {
+  setenv("PATH", PATH.c_str(), /*override*/ 1);
+}
+static std::string addToPATH(std::string newPathElm) {
+  std::string curPath = std::getenv("PATH");
+  std::string adjPath = curPath + std::string(":") + newPathElm;
```

Comment:
I think it's debatable whether `newPathElm` should go to the front or end of the path.

As I understand it in Bash (and I assume other shells?) the path is processed left-to-right so if the user had an earlier directory with `ptxas` (or whatever) that would get picked up first. Which may or may not be what they want?

Reply:
I put it at the end because thats what we do for the AMD LLVM